avm/ptn/deployment-script/import-image-to-acr/tests/e2e/max/main.test.bicep

```diff
@@ -46,6 +47,11 @@ resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
 // Test Execution //
 // ============== //
 
+resource keyVault 'Microsoft.KeyVault/vaults@2023-07-01' existing = {
```

Comment:
I take it there is no warning for this resoure? Could've imagined as it's not used wih the testing being commented out.

Is the reason you'd still want to keep it in to enable you to manually test it?

Reply:
No warning, right. It will be needed in the test for the authentication data which is stored there. This occurrence only fetches the existing Key Vault. It is created in the dependencies. We could remove it entirely, as it is only used if uncommented. But I'd like to keep it this way for documentation/example purposes.